agent/src/main.py

```
@@ -425,6 +444,221 @@ def list_pending_discoveries(_user_id: str = Depends(require_auth)):
+        .order("created_at", desc=True)
+        .limit(100)
+        .execute()
+    )
+
+    actions = []
+    for disc in disc_resp.data:
+        proj = disc.get("project") or {}
+        epc_name = disc.get("epc_contractor", "")
+
+        # Look up entity + contacts
+        entity = resolve_entity(epc_name) if epc_name else None
```

Ruff (0.15.9)

[error] Undefined name `resolve_entity`

(F821)